nmrglue/process/proc_autophase.py

```diff
@@ -68,35 +68,42 @@ def _ps_acme_score(ph, data):
         Value of the objective function (phase score)
 
     """
-    stepsize = 1
-
+ """
```

Comment:
I think this is why the Travis CI tests are failing. The ending ` """ ` in the doctoring should be indented by 4 spaces.